### PR TITLE
Update project name

### PR DESCRIPTION
**Pull request by rkondratowicz**
in gradle na terraform

### CODE REVIEW

File: infrastructure/variables.tf [hunk 1]
```diff
@@ -1,11 +1,11 @@
 variable "product" {
   type    = "string"
-  default = "send-letter"
+  default = "send-letter-service"
 }
 
 variable "microservice" {
   type = "string"
-  default = "producer"
+  default = "api"
```
Comment (doncem) [inline infrastructure/variables.tf]: is it api and not service?

**Comment by doncem** [comment]
still no jenkins file 🙃

File: infrastructure/variables.tf [hunk 2]
```diff
@@ -5,7 +5,7 @@ variable "product" {
 
 variable "microservice" {
   type = "string"
-  default = "api"
+  default = "service"
```
Comment (nitinprabhu) [inline infrastructure/variables.tf]: This doesn't look right to me micro service as service ?I would expect some thing similar to 
`"service-auth-provider` or 'pdf-service`

File: infrastructure/main.tf [hunk 3]
```diff
@@ -37,14 +37,14 @@ resource "vault_generic_secret" "servicebus-listen-conn-string" {
 
 module "db" {
   source              = "git@github.com:contino/moj-module-postgres.git?ref=feature/specify-db-name"
```
Comment (lgonczar) [inline infrastructure/main.tf]: The branch must be master and postgresql_database must be `postgres`, unfortunately

File: infrastructure/main.tf [hunk 4]
```diff
@@ -37,14 +37,14 @@ resource "vault_generic_secret" "servicebus-listen-conn-string" {
...
-  product             = "${var.product}"
+  product             = "${var.product}-db"
```
Comment (lgonczar) [inline infrastructure/main.tf]: Just so you know, postgres module will add `-data` to the name.

**Pull request merged**